standup/settings.py

```diff
@@ -144,11 +144,17 @@ def path(*paths):
 SECURE_BROWSER_XSS_FILTER = config('SECURE_BROWSER_XSS_FILTER', default='true', parser=bool)
 # this should be 31536000 in prod (1 year)
 SECURE_HSTS_SECONDS = config('SECURE_HSTS_SECONDS', default='0', parser=int)
+SECURE_HSTS_INCLUDE_SUBDOMAINS = config('SECURE_HSTS_INCLUDE_SUBDOMAINS',
+                                        default='false', parser=bool)
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 SECURE_SSL_REDIRECT = config('SECURE_SSL_REDIRECT', default='false', parser=bool)
+SESSION_COOKIE_SECURE = CSRF_COOKIE_SECURE = SECURE_SSL_REDIRECT
 
-GRAVATAR_DEFAULT_IMAGE = 'http://www.standu.ps/static/img/default-avatar.png'
-GRAVATAR_DEFAULT_SECURE = False
+# must be a full publically accessible URL, or a name of a gravatar default theme
+GRAVATAR_DEFAULT_IMAGE = 'https://raw.githubusercontent.com/mozilla/standup/master/standup/' \
+                         'status/static/img/default-avatar.png'
```

Comment:
So... this is a python file and if we know by this point whether the site is https and hsts, then we can know which default url to use.

Oh, wait... Do we not know the protocol and host for this instance, yet? If not, we could add that as a setting and then key off of that. Seems like it'd help to know it possibly in other places, too.